src/convertToObject.js

```diff
@@ -3,16 +3,34 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related
+ * CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  let styles = sourceString.replace(/\n*/g, '').split(';');
+
+  styles.forEach((el, i) => {
+    styles[i] = el.trim();
+
+    if (styles[i].length === 0) {
+    }
+  });
```

Comment:
can you provide explanation to this loop? I can't wrap my head around it 🤔

Reply:
a trimmed all lines, to make empty lines with 0 length (to filter it in next line). It's not necessary - I have added trim to filter in next line and it also works